packages/client/src/raffles/types/patchRaffleParticipation.types.ts

```diff
@@ -0,0 +1,16 @@
+import type { Config } from '../..';
+import type { Raffle } from './raffles.types';
+
+export type PatchRaffleParticipationData = {
+  value: string;
```

Comment:
Review this type. It appears to be an object and not a string.

Reply:
Nobody knows which fields can be updated. I spoke with Pedro Ramos, who implemented the raffles in Sakai, and he sent me the example below: 
 
<img width="448" alt="Screenshot 2023-01-30 at 19 39 34" src="https://user-images.githubusercontent.com/38541271/215578552-8f48661a-c9fd-4b56-a415-aeb162a46ffe.png">

Reply:
That endpoint uses the [JSON Patch](https://jsonpatch.com/) format (several PATCH endpoints are already being created like this)

Reply:
Also to note that this `data` is also an array of operations and not a single object.

Reply:
done

Reply:
The file hasn't changed.

Reply:
This line hasn't changed. Value could other types other than `string` .

Reply:
Should I change the typed string to unknow?

Reply:
Updated to unknown

Reply:
Please double check. I believe `value` and `from` for `Operation` are [optional](https://ffdevelopers.farfetch.tech/api/commerce/def/operation.html).

Reply:
If it is based on JSON Patch standard, then you can install the `@types/json-patch` package and use the types defined there are you can see [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/json-patch/index.d.ts) (Use the `OpPatch` type and make sure you have it in `dependencies` and not in `devDependencies` since it is a part of the public exports)

Reply:
Took the liberty to update the PR to the use `@types/json-patch` package to define the types for not only this endpoint but all endpoints that accept values based on the json patch schema.